src/__tests__/ariaAttributes.js

```diff
@@ -9,6 +9,11 @@ test('`selected` throws on unsupported roles', () => {
   )
 })
 
+test('`selected: true` matches `checked` checkboxes', () => {
+  const {getByRole} = renderIntoDocument(`<input type="checkbox" checked />`)
+  expect(getByRole('checkbox', {selected: true})).toBeInTheDocument()
+})
```

Comment:
Could we get a test for the aria-checked version as well?

And it may help to make this test more complete if we include a second checkbox that is not checked.